src/internal/git/pull.go

```diff
@@ -54,7 +56,19 @@ func pull(gitUrl string, targetFolder string, spinner *message.Spinner) {
 	if err == git.ErrRepositoryAlreadyExists {
 		spinner.Debugf("Repo already cloned")
 	} else if err != nil {
```

Comment:
Is there a specific err type that we can check for instead of catching all of them?

Or at least, we should debug log the original err. If there is an err with go-git cloning the repo and the user doesn't have `git` in their path then the messing logged to debug will not be (as?) useful.

Reply:
There are other errors we _could_ catch here, but I think it's cleaner to fall-through on any error for now because the intent is to get the git data one way or another and there will likely be other edge cases we haven't identified yet.